packages/web/src/views/pool/pool-add/components/additional-info/exchange-rate-graph/pair-ratio/PairRatio.tsx

```diff
@@ -9,8 +9,8 @@ import { pulseSkeletonStyle } from "@constants/skeleton.constant";
 import { PoolModel } from "@models/pool/pool-model";
 import { formatTokenExchangeRate } from "@utils/stake-position-utils";
 
-import IconSwap from "../icons/IconSwap";
-import MissingLogo from "../missing-logo/MissingLogo";
+import IconSwap from "../../../../../../../components/common/icons/IconSwap";
+import MissingLogo from "../../../../../../../components/common/missing-logo/MissingLogo";
```

Comment:
What about importing using prefixes ?

Reply:
fixed in 7bbebee2ada9ea280036b96ce17941cc7253bc17